styles.css

```diff
@@ -418,3 +426,469 @@ body {
+    font-size: 15px;
+    line-height: 20px;
+    max-width: 80%;
+    text-align: center;
+    margin-top: 16px;
+    color: #3a4a42;
+    margin-bottom: 10px;
+  }
+
+  .card-head {
+    margin-bottom: 50px;
+  }
+
+  .tags-list {
+    margin: 10px 0;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .tag {
+    padding: 8px;
+    background: #ebf0ee;
+    border-radius: 4px;
+    margin: 5px;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 9px;
+    letter-spacing: 0.03em;
+    color: #3a4a42;
+  }
+
+  .about-me {
+    margin-top: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    margin-bottom: -30px;
+  }
+
+  .about-me-text {
+    background-image: url("images/aboutme-desktop.svg");
+    background-position: bottom left;
+    background-repeat: no-repeat;
+  }
+
+  .btn-abt-desk {
+    margin-top: 30px;
+    padding: 1px !important;
+    font-size: 14px;
+  }
+
+  .bg-about {
+    background-image: url(images/bg-abt.png);
+    background-size: 20%;
+    background-repeat: no-repeat;
+    background-position: top right;
+  }
+
+  .supporting-text-about {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 28px;
+    text-align: center;
+    color: #172b4d;
+    padding-left: 260px;
+    padding-right: 260px;
+    margin-bottom: 30px;
+  }
+
+  .tools-list {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    margin-top: 40px;
+  }
+
+  .tools {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+  }
+
+  .tool-card {
+    margin: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    background: #ebf0ee;
+    padding: 20px;
+    box-sizing: border-box;
+    border-radius: 8px;
+    width: 330px;
+    height: 332px;
+    border: 1px solid #c1c7d0;
+  }
+
+  .card-icon {
+    display: flex;
+    width: 80px;
+    height: 80px;
+    border-radius: 50%;
+    background: #fff;
+    justify-content: center;
+    align-items: center;
+    box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+  }
+
+  .icon {
+    width: 48px;
+    height: 48px;
+  }
+
+  .tool-tags {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 5px;
+    flex-wrap: wrap;
+    max-width: 327px;
+  }
+
+  .tool-tag {
+    padding: 8px 12px;
+    margin: 0 4px 10px;
+    background: #fff;
+    border-radius: 8px;
+    font-weight: 600;
+    align-self: center;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #36b37e;
+  }
+
+  .Contact-me {
+    padding: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 700px;
+  }
+
+  .form-flex {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
```

Comment:
- [ ] Please add the right width for The form and the heading text beside it and edit the fint size and wieght for that text too. and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159703113-6a995c8f-1efb-49b1-845e-9e5dd2b7ced4.png) | ![image](https://user-images.githubusercontent.com/68796232/159703587-eb0048f7-1d69-47de-adb7-1946de11899f.png) |

Reply:
All good now 👍🏽